app/helpers/attachments_helper.rb

```diff
@@ -1,4 +1,8 @@
 module AttachmentsHelper
+  # The following departments are taking place in a pilot scheme to use the accessible format
+  # request form. DFE(6), DWP(10), DHSC(12), HMRC(25), DVSA(570), UKHSA(1328)
+  ACCESSIBLE_FORMAT_REQUEST_PILOT_ORGANISATION_IDS = [6, 10, 12, 25, 570, 1328].freeze
```

Comment:
We could fetch the email addresses from GOVUK Publishing Components and then use them to query the organisation, which would reduce the need to maintain two lists. We'd just need to pass the email instead of org_id from the view:

```
address = attachment.attachable.alternative_format_provider.alternative_format_contact_email

addresses = GovukPublishingComponents::Presenters::Attachment::EMAILS_IN_ACCESSIBLE_FORMAT_REQUEST_PILOT

addresses.include?(address)
```

Reply:
good idea, implemented this now the publishing component have been updated